eip-0019.md

```diff
@@ -175,16 +175,33 @@ For better security and usability additional data can be transfered between Hot
 and Cold wallets via QR codes. Hot Wallet passes data to the Cold Wallet using
 ColdSigningRequest data format and Cold Wallet replies back with ColdSigningResponse.
 
-```
-ColdSigningRequest:
- - messageType: Byte = 1
- - transaction: ReducedTransaction
- - description: String
-
-ColdSigningResponse:
- - messageType: Byte = 2
- - transaction: SignedTransaction
-```
+### ColdSigningRequest
+Json format, holding
+* reducedTx (mandatory): Base64-encoded `ReducedTransaction` as defined above
+* sender (optional): P2PK address sending the transaction, can be used by cold wallets to determine which secret to use for signing the transaction
```

Comment:
In the case of EIP3 addresses, many secret keys may need to be added to the prover in order to sign the transaction.
So, it is better to make this field a list (json array).